server/src/graphql/resolvers/user.ts

```
@@ -1,123 +1,438 @@
+          'SELECT id FROM users WHERE email = $1 OR username = $2',
+          [input.email.toLowerCase(), input.username]
+        );
+
+        if (existingUser.rows.length > 0) {
+          throw new GraphQLError('User with this email or username already exists', {
+            extensions: { code: 'USER_EXISTS' },
+          });
+        }
+
+        // Generate a temporary password
+        const tempPassword = require('crypto').randomBytes(16).toString('hex');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)